docs/products/clickhouse/howto/connect-with-python.rst

```diff
@@ -0,0 +1,95 @@
+Variable                Description
+==================      =====================================================================
+``USERNAME``            ``User`` for the ClickHouse connection available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse native**
+``PASSOWRD``            ``Password`` for the ClickHouse connection available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse native**
+``HOST``                ``Host`` for the ClickHouse connection available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse native**
+``NATIVE_PORT``         ``Port`` for the ClickHouse connection available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse native**
+``query``               Query you want to run, for example ``SELECT 1``
+==================      =====================================================================
+
+Connect to the service
+''''''''''''''''''''''
+
+Replace the placeholders in the code with meaningful information on your service connection and run the code.
+
+.. code:: python
+
+    from clickhouse_driver import Client
+    client = Client(user="USERNAME", password="PASSWORD", host="HOST", port=NATIVE_PORT, secure=True)
+    print(client.execute("SELECT 1"))
+
+Connect with HTTPS
+------------------
+
+Pre-requisites
+''''''''''''''
+
+* `Python 3.7 or later <https://www.python.org/downloads/>`_
+
+* `Requests HTTP library <https://pypi.org/project/requests/>`_
+
+Identify connection information
+'''''''''''''''''''''''''''''''
+
+To run the code for connecting to your service, first identify values of the following variables:
+
+===========================     ======================================================================================
+Variable                        Description
+===========================     ======================================================================================
+``https://HOST:HTTPS_PORT``     ``Host`` and ``Port`` for the ClickHouse connection available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse HTTPS & JDBC**
+``query``                       Query you want to run, for example ``SELECT 1``
+``X-ClickHouse-Database``       ``Database Name`` available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse HTTPS & JDBC**, for example ``system``
+``X-ClickHouse-User``           ``User`` for the ClickHouse connection available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse HTTPS & JDBC**
+``X-ClickHouse-Key``            ``Password`` for the ClickHouse connection available in the Aiven console: Service **Overview** > **Connection information** > **ClickHouse HTTPS & JDBC**
+``X-ClickHouse-Format``         Format for the output from your query, for example ``JSONCompact``
```

Comment:
The available formats the same as in https://docs.aiven.io/docs/products/clickhouse/reference/supported-input-output-formats.html? 
BTW, on that note, the title for https://docs.aiven.io/docs/products/clickhouse/reference/supported-input-output-formats.html was changed to "Formats for ClickHouse-Kafka data exchange". I believed that these formats are not only Kafka related, but general ones. You can see here the article on the side of  ClickHouse docs - https://clickhouse.com/docs/en/interfaces/formats/

Reply:
@anelook , the title has been changed as requested in https://aiven.atlassian.net/browse/DOC-20 by @kmichel-aiven

Reply:
Yes :)

The formats available to connect ClickHouse with python are all of these: https://clickhouse.com/docs/en/interfaces/formats/
The formats available to connect ClickHouse with Kafka are only a small subset: the ones in https://docs.aiven.io/docs/products/clickhouse/reference/supported-input-output-formats.html